git-hub

```diff
@@ -294,19 +294,30 @@ def git_dir():
 # The contents of .git/HUB_EDITMSG are returned by the function (after the
 # editor is closed). The text is not filtered at all at this stage (so if the
 # user left the `help_msg`, it will be part of the returned string too.
-def editor(help_msg, msg=None):
+def editor(help_msg, msg=""):
```

Comment:
It was previously important to distinguish between an user-provided empty message and no message provided at all. Not sure if this is still relevant, are you sure it is OK to change this?